### PR TITLE
Add new idea for LibreMesh internet control

### PR DESCRIPTION
**Pull request by luandro**
### Checklist for new or updates project ideas

- [x] I added at least one contact option so interested people can get in touch with me.
- [x] Data for my project is valid

### CODE REVIEW

File: _projects/libremesh-internet-control.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+difficulty: high
+status: open
+initiatives:
+- GSoC 2024
+issues:
+- "Difficulty in tracking and whitelisting services like WhatsApp due to changing IPs"
+size: 350 hours
+markdown: libremesh-internet-control.md
+mentors:
+- name: mentor from the LibreMesh community
```
Comment (andibraeu) [inline _projects/libremesh-internet-control.md:19]: please give some more details about who will be the mentors and how to contact them

File: _projects/libremesh-internet-control.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Intervoices
+developers_involved:
+- technician from the Guarani community
+- LibreMesh developers
+difficulty: high
+status: open
+initiatives:
+- GSoC 2024
+issues:
+- "Difficulty in tracking and whitelisting services like WhatsApp due to changing IPs"
```
Comment (andibraeu) [inline _projects/libremesh-internet-control.md:15]: our thoughts behind this field were to add links to github issues or other issue trackers

File: _projects/libremesh-internet-control.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+mentors:
+- name: mentor from the LibreMesh community
+  contact:
+    github: mentor_github_handle
+    email: mentor_email_address
+    other_chat_options:
+    - mentor_matrix_link
+requirements:
+- Networking in mesh networks
+- Experience with IP tables and traffic control
```
Comment (andibraeu) [inline _projects/libremesh-internet-control.md:27]: what kind of coding skills do contributors need?

**Review by andibraeu — CHANGES_REQUESTED** [review]

**Comment by luandro** [comment]
Thanks for the comments @andibraeu, I was just adding more details... but ur comments also cleared some things up.

**Comment by andibraeu** [comment]
thanks for your quick response. now you somehow included some older commits in this PR. Can you please update your branch and force-push your changes?

**Comment by andibraeu** [comment]
there should also be no changes to Gemfile.lock (renovate will take care of these)

**Comment by luandro** [comment]
> thanks for your quick response. now you somehow included some older commits in this PR. Can you please update your branch and force-push your changes?

Oh weird, yea, will try to remove those.

**Comment by luandro** [comment]
Git wouldn't behave, so opened new #50

**Pull request closed without merging**